com.github.bleakgrey.tootle.json

```diff
@@ -0,0 +1,56 @@
+{
+  "app-id": "com.github.bleakgrey.tootle",
+  "runtime": "org.gnome.Platform",
+  "sdk": "org.gnome.Sdk",
+  "runtime-version": "3.36",
```

Comment:
3.38

Reply:
Erf I forgot I was on my Flatpak branch… dropped that commit.

Reply:
For your curiosity, the 3.38 runtime was because somehow both master and 3.38 were broken, that's also why I added libhandy…